packages/react/src/components/TimePickerSpinner/__snapshots__/TimePickerSpinner.story.storyshot

```diff
@@ -1,356 +1,5 @@
-          >
-            <input
-              aria-label="Pick a time"
-              className="bx--time-picker__input-field bx--text-input"
-              data-testid="time-picker-spinner"
-              disabled={false}
-              id="timepicker"
-              maxLength={5}
-              onBlur={[Function]}
-              onChange={[Function]}
-              onClick={[Function]}
-              onKeyDown={[Function]}
-              onKeyUp={[Function]}
-              pattern="(1[012]|[1-9]):[0-5][0-9](\\\\s)?"
-              placeholder="hh:mm"
-              type="text"
-              value="19:33"
-            />
-          </div>
-          <div
-            className="iot--time-picker__controls"
-          >
-            <button
-              aria-atomic="true"
-              aria-label="Increment minutes"
-              aria-live="polite"
-              className="iot--time-picker__controls--btn up-icon"
-              data-testid="time-picker-spinner-up-button"
-              disabled={false}
-              onBlur={[Function]}
-              onClick={[Function]}
-              onFocus={[Function]}
-              onMouseOut={[Function]}
-              onMouseOver={[Function]}
-              title="Increment minutes"
-              type="button"
-            >
-              <svg
-                aria-hidden={true}
-                className="up-icon"
-                fill="currentColor"
-                focusable="false"
-                height={4}
-                preserveAspectRatio="xMidYMid meet"
-                viewBox="0 0 8 4"
-                width={8}
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  d="M0 4L4 0 8 4z"
-                />
-              </svg>
-            </button>
-            <button
-              aria-atomic="true"
-              aria-label="Decrement minutes"
-              aria-live="polite"
-              className="iot--time-picker__controls--btn down-icon"
-              data-testid="time-picker-spinner-down-button"
-              disabled={false}
-              onBlur={[Function]}
-              onClick={[Function]}
-              onFocus={[Function]}
-              onMouseOut={[Function]}
-              onMouseOver={[Function]}
-              title="Decrement minutes"
-              type="button"
-            >
-              <svg
-                aria-hidden={true}
-                className="down-icon"
-                fill="currentColor"
-                focusable="false"
-                height={4}
-                preserveAspectRatio="xMidYMid meet"
-                viewBox="0 0 8 4"
-                width={8}
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  d="M8 0L4 4 0 0z"
-                />
-              </svg>
-            </button>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-`;
-
```

Comment:
why this got updated ?

Reply:
+1. Not sure why this would be affected.

Reply:
Not sure myself

Reply:
![image](https://user-images.githubusercontent.com/18572477/188665680-429bf122-9257-4806-b520-e57b2f100513.png)
Looks like there was some space. But its fine.